app/models/user.rb

```diff
@@ -0,0 +1,8 @@
+class User < ApplicationRecord
+  has_many :posts
+  has_many :comments
+  has_many :likes
```

Comment:
- [ ] Kindly make sure to add foreign_keys for the has_many relationship e.g `has_many :posts, foreign_key: 'author_id'`